pilot/pkg/config/clusterregistry/conversion.go

```diff
@@ -16,11 +16,9 @@ package clusterregistry
 
 import (
 	"fmt"
-	"strconv"
 	"strings"
 
-	multierror "github.com/hashicorp/go-multierror"
-	"go.uber.org/multierr"
+	"github.com/hashicorp/go-multierror"
```

Comment:
lint doesn't like it, but the following seems to make it happy

multierror "github.com/hashicorp/go-multierror"